packages/skeleton-svelte/src/lib/components/Tab/TabsControl.svelte

```diff
@@ -0,0 +1,126 @@
+<script lang="ts">
+	import { getContext } from 'svelte';
+	import type { TabsControlProps } from './types.js';
+
+	let {
+		name,
+		group,
+		title = '',
+		// A11y
+		controls = '',
+		// Root
+		base = getContext<string>('base'),
+		text = getContext<string>('text'),
+		justify = getContext<string>('justify'),
+		active = getContext<string>('active'),
+		inactive = getContext<string>('inactive'),
+		background = getContext<string>('background'),
+		padding = getContext<string>('padding'),
+		rounded = getContext<string>('rounded'),
+		spacingY = getContext<string>('spacingY'),
+		cursor = getContext<string>('cursor'),
+		classes = getContext<string>('classes'),
+		// Tab
+		tabBase = '',
+		tabRounded = 'rounded-container',
+		tabOutline = 'focus:outline focus:outline-primary-600-400 focus:outline-offset-4',
+		tabClasses = '',
+		// Tab Content
+		contentBase = 'flex',
+		contentSpaceX = 'space-x-1',
+		contentClasses = '',
+		// Events
+		onclick = getContext<(event: MouseEvent) => {}>('onclick'),
+		onkeypress = getContext<(event: KeyboardEvent) => {}>('onkeypress'),
+		onkeydown = getContext<(event: KeyboardEvent) => {}>('onkeydown'),
+		onkeyup = getContext<(event: KeyboardEvent) => {}>('onkeyup'),
+		onchange = getContext<(group: string) => {}>('onchange'),
+		// Snippets
+		children
+	}: TabsControlProps = $props();
+
+	const selected = $derived(group === name);
+	const rxActive = $derived(selected ? active : inactive);
+
+	let elemInput: HTMLInputElement;
+
+	function onTabKeyDown(event: KeyboardEvent) {
+		onkeydown(event);
+		if (!['Enter', 'Space', 'ArrowRight', 'ArrowLeft', 'Home', 'End'].includes(event.code)) return;
+
+		event.preventDefault();
```

Comment:
Avoid this in the global event handler. Keep it localized only when overwriting very specific behavior. This can inadvertently damage built in features like enter/space to select.